test/unit/firebase.js

```diff
@@ -36,6 +36,29 @@ describe('MockFirebase', function () {
     });
   });
 
+  describe('Array rendering', function () {
+    it('renders integer–keyed objects as arrays', function () {
```

Comment:
Not all outputs are valid.  If the keys are not sequential, the returned data from firebase is an object.  See the comment I added https://github.com/soumak77/firebase-mock/issues/35#issuecomment-323888463

Reply:
Looks like it handles those examples correctly. Perhaps the name of the test is misleading? s/integer-keyed/array-like/?

Reply:
Among others, this test is wrong:

`{input: {0: 'a', 2: 'c', 3: 'd', 4: 'e'}, output: ['a', null, 'c', 'd', 'e']}`

The output should be the same as the input.